client/state/README.md

```diff
@@ -96,15 +108,19 @@ We are provided the opportunity to make straightforward tests without complicate
 #### Example
 
 ```js
-const age = ( state = 0, action ) =>
-    GROW === action.type
-        ? state + 1
-        : state
+function age( state = 0, action ) {
+	if ( GROW === action.type ) {
+		return state + 1 
+	}
+	return state;
+}
```

Comment:
any specific reason to change these?

Reply:
> modify some documentation functions to be friendlier to newcomers (remove ternaries, use classic function over arrow functions)

I can change them back if you prefer!

Reply:
Mainly I'd rather have the changes as a separate PR with a reason for changing them since it didn't seem at all related to this PR, or that this PR may not be the most appropriate place to discuss the change.

Reply:
very fair. i'll revert